### PR TITLE
timerfd: Add unit tests

### LINKED ISSUE #25: timerfd.rs - add unit tests
The coverage as of 30 July 2019 is 67.9%.

The following functions and trait implementations are not tested:
- `clear()`
- `FromRawFd`
- `IntoRawFd`

Also, error cases are also not tested, but I am not sure how feasible it is to test them.

### PR DESCRIPTION
**Pull request by liujing2**
clear(), FromRawFd and IntoRawFd are tested.

Fix #25.

Signed-off-by: Jing Liu <jing2.liu@linux.intel.com>

### CODE REVIEW

**Comment by jiangliu** [comment]
Need to update coverage_config.json to fix the failure:
AssertionError: Coverage is increased from 67.9 to 69.6. Please update the coverage in tests/coverage.

**Comment by liujing2** [comment]
> Need to update coverage_config.json to fix the failure:
> AssertionError: Coverage is increased from 67.9 to 69.6. Please update the coverage in tests/coverage.

Thanks for pointing out. Updated now.

File: src/timerfd.rs [hunk 1]
```diff
@@ -233,6 +246,7 @@ mod tests {
 
         assert_eq!(count, 1);
         assert!(now.elapsed() >= dur);
+        tfd.clear().expect("unable to clear the timer");
```
Comment (andreeaflorescu) [inline src/timerfd.rs]: I don't think that calling the clear function in this case is enough. You should also check that the `clear` function had the expected effect.

Comment (liujing2) [reply src/timerfd.rs]: Fixed.

**Pull request merged**